rocAL/include/readers/image/mxnet_recordio_reader.h

```diff
@@ -76,31 +76,20 @@ class MXNetRecordIOReader : public Reader {
     std::string _image_key;
     std::vector<std::string> _file_names;
     std::map<std::string, std::tuple<unsigned int, int64_t, int64_t>> _record_properties;
-    unsigned _curr_file_idx;
     unsigned _current_file_size;
     std::string _last_id, _last_file_name;
     unsigned int _last_file_size;
     int64_t _last_seek_pos;
     int64_t _last_data_size;
-    size_t _shard_id = 0;
-    size_t _shard_count = 1;  // equivalent of batch size
-    //!< _batch_count Defines the quantum count of the images to be read. It's usually equal to the user's batch size.
-    /// The loader will repeat images if necessary to be able to have images available in multiples of the load_batch_count,
-    /// for instance if there are 10 images in the dataset and _batch_count is 3, the loader repeats 2 images as if there are 12 images available.
-    size_t _batch_count = 1;
+    size_t _batch_size = 1;
```

Comment:
why is the explanation removed for batch_size

Reply:
Previously, the explanation was for batch count which has now been modified to batch size.
Now the images in the loader will be repeated or dropped based on the Last batch policy and not dependent on the batch count.